samples/config.default.js

```diff
@@ -1,36 +1,57 @@
-// User-specific configuration
-exports.voice = 'en-US_MichaelVoice';
-exports.conversationWorkspaceId = ''; // replace with the workspace identifier of your conversation
+// User-specific CONFIGURATION
+exports.config = {}
+exports.config.voice = {"gender": "female", "language": "en-US"};
```

Comment:
Just gender and language.
TJ at the initialization will get all voices available in Text to Speech (it could be change with time) and select the best voice to that gender and language.

It is also used in Speech to Text instead of previous default "en-US_BroadbandModel".
In previous version TJ did understand nothing if you speak in other language than english.
Now he understand, write the correct words in other language but do nothing because of missing recipes.